### ISSUE TITLE
ReferenceError: $_$wp is not defined

### ISSUE THREAD

**Issue by pahund** [wallabyjs/public#1893]
### Issue description or question

*Note:* I know this has been reported before, and allegedly fixed with a new Wallaby version, but I do use the latest version, and still get the error.

I'm using the latest Enzyme version 3.7.0, React 16 Adapter 1.6.0 with React 16.6.0.

When I start Wallaby in IntelliJ IDEA, all the tests fail with error message: 
> ReferenceError: $_$wp is not defined

According to Wallaby's in-editor error log display, the error occurs when I try to configure Enzyme with the React 16 adapter in my `setup-jest.js` module:

![image](https://user-images.githubusercontent.com/1639313/47707768-ed238580-dc2c-11e8-94a0-2090fdbbbb7e.png)

The stack trace shown on Wallaby's console:

```
node_modules/jest-runner/node_modules/jest-jasmine2/build/index.js:122:17)
    at /Users/pahund/git/polly/node_modules/jest-runner/build/run_test.js:73:28
    at Generator.next (<anonymous>)
    at step (/Users/pahund/git/polly/node_modules/jest-runner/build/run_test.js:147:191)
    at /Users/pahund/git/polly/node_modules/jest-runner/build/run_test.js:147:361
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Runtime error: ReferenceError: $_$wp is not defined
    at Object.<anonymous> (./scripts/utils/setup-jest.js:3:1)
    at Runtime._execModule (/Users/pahund/git/polly/node_modules/jest-runtime/build/index.js:448:13)
    at Runtime.requireModule (/Users/pahund/git/polly/node_modules/jest-runtime/build/index.js:265:14)
    at /Users/pahund/git/polly/node_modules/jest-runner/node_modules/jest-jasmine2/build/index.js:80:15
    at Generator.next (<anonymous>)
    at step (/Users/pahund/git/polly/node_modules/jest-runner/node_modules/jest-jasmine2/build/index.js:144:191)
    at /Users/pahund/git/polly/node_modules/jest-runner/node_modules/jest-jasmine2/build/index.js:144:437
    at new Promise (<anonymous>)
    at /Users/pahund/git/polly/node_modules/jest-runner/node_modules/jest-jasmine2/build/index.js:144:99
    at jasmine2 (/Users/pahund/git/polly/node_modules/jest-runner/node_modules/jest-jasmine2/build/index.js:122:17)
    at /Users/pahund/git/polly/node_modules/jest-runner/build/run_test.js:73:28
    at Generator.next (<anonymous>)
    at step (/Users/pahund/git/polly/node_modules/jest-runner/build/run_test.js:147:191)
    at /Users/pahund/git/polly/node_modules/jest-runner/build/run_test.js:147:361
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

### Wallaby.js configuration file

```javascript
process.env.BABEL_ENV = 'test';

const { version: softwareVersion } = require('./package.json');

module.exports = wallaby => ({
    files: [
        'package.json',
        'scripts/utils/setup-jest.js',
        'src/**/*.js',
        'src/**/*.css',
        'src/**/*.json',
        '!src/**/*.test.js',
        '!src/**/*.testx.js'
    ],

    tests: ['src/**/*.test.js', '!node_modules/**'],

    env: {
        type: 'node',
        runner: 'node',
        params: {
            // this allows us to use absolute paths in import statements instead of ../../../../../bla.js
            env: `NODE_PATH=${wallaby.projectCacheDir};NODE_ENV=test;npm_package_version=${softwareVersion}`
        }
    },

    testFramework: 'jest',

    compilers: {
        '**/*.js': wallaby.compilers.babel()
    },

    reportUnhandledPromises: false
});
```

### Code editor or IDE name and version

IntelliJ IDEA 2018.2.5 (Ultimate Edition)
Build #IU-182.4892.20, built on October 16, 2018

Wallaby.js core v1.0.630

### OS name and version

macOS High Sierra 10.13.6

**Comment by pahund** [comment]
@ArtemGovorov sure, here's my complete `package.json` (also including Jest config): https://gist.github.com/pahund/152e9e01fb17268f125603345f8a84b3

**Comment by ArtemGovorov** [comment]
Thanks for providing the config!

Unfortunately I wasn't able to reproduce the issue yet. I have created this sample repo https://github.com/ArtemGovorov/wallaby-1893 with your package.json and Wallaby config, but it seems to work for me. I had to exclude a few dependencies that are not publicly available though.

Could you please 
- try the sample repo to see if Wallaby is working for you?
- if it is working, could you please share the result of running `npm ls jest-util`?

My guess is that possibly multiple different versions of Jest/or parts of Jest are somehow installed in your project (and that confuses Wallaby), but I couldn't verify it because of the private dependencies that I'm missing.

**Comment by pahund** [comment]
At first I got the same error *$_$wp is not defined* when trying to run the sample repo you have prepared.

**But** your remark about Wallaby being confused about multiple Jest versions got me thinking, so I examined my home directory, and sure enough, there was a *node_modules* dir containing the Jest package there!

I must have accidentally installed Jest in my home dir at some point. After deleting that *node_modules* directory in the home dir, the sample repo runs fine, and so do all my other projects that use Wallaby.

👍🏻 Thanks for your quick help!
